JPF/bad_buildings.sql

```diff
@@ -455,10 +556,75 @@ left join
 	aep_2015 a15 on (a15.full_addr) = (hbaddress)		
 left join
 	aep_2016 a16 on (a16.full_addr) = (hbaddress)
+left join
+	hpd_buildings_temp hb on hb.buildingid = cc.buildingid
 ;
 
+## CREATE AND SET THE TARGET COLUMN
+ALTER TABLE bad_buildings ADD `target` INT  NULL  DEFAULT NULL  AFTER `managementprogram`;
+ALTER TABLE bad_buildings ADD `bb_target` INT  NULL  DEFAULT NULL  AFTER `target`;
+ALTER TABLE bad_buildings ADD `bb_score` FLOAT  NULL  DEFAULT NULL  AFTER `bb_target`;
+alter table bad_buildings add index(unitsres);
+alter table bad_buildings add index(pa_score);
+alter table bad_buildings add index(dob_violation_cnt);
+alter table bad_buildings add index(total_aep_yrs);
```

Comment:
These ALTER TABLE statements might be able to be combined into a single statement, with sub-statements separated by commas.